BoxContentSDK/BoxContentSDK/Operations/BOXAPIAuthenticatedOperation.m

```diff
@@ -78,7 +77,7 @@ - (void)processResponse:(NSURLResponse *)response
         {
             self.error = [[NSError alloc] initWithDomain:BOXContentSDKErrorDomain code:BOXContentSDKAuthErrorAccessTokenExpiredOperationWillBeClonedAndReenqueued userInfo:nil];
 
-            BOXAPIJSONOperation *operationCopy = [self copy];
```

Comment:
... And then using using that subclass here. `self` isn't always a BOXAPIJSONOperation instance. It can be any subclass that returns `YES` from `canBeReenqueued:`. Yikes.